prow/plugins/trigger/ic.go

```diff
@@ -26,8 +26,14 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
-var okToTest = regexp.MustCompile(`(?m)^(@k8s-bot )?ok to test\r?$`)
-var retest = regexp.MustCompile(`(?m)^/retest\s*$`)
+// The commands can have additional commands behind them, but not before
+// Any additional comment and white spaces before the commands will NOT trigger the commands
+// The commands can have comma appended, e.g., "ok to test," "/ok-to-test," "/test," "/test unit,"
+var oldRegex = `(?m)^(@k8s-bot )?ok to test,?(\s+|$)`
+var newRegex = `(?m)^/ok-to-test,?(\s+|$)`
+var regex = fmt.Sprintf("(%s|%s)", oldRegex, newRegex)
+var okToTest = regexp.MustCompile(regex)
```

Comment:
as we will not support trailing whitespaces, comma, and words for `ok-to-test`, a line suffices.